backend/pyproject.toml

```diff
@@ -8,14 +8,13 @@ packages = [{include = "app", from="."}]
 
 [tool.poetry.dependencies]
 python = "^3.9"
-fastapi = "^0.68.1"
+fastapi = "^0.79.1"
 uvicorn = "^0.15.0"
 aiofiles = "^0.4.0"
 invoke = "^1.4.1"
-httpx = "^0.19.0"
+httpx = "^0.23.0"
 mock = "^4.0.2"
 python-multipart = "^0.0.5"
-pyjwt = "^2.1.0"
```

Comment:
Since we in the code actually use (py)jwt directly (not only from pyjwt-key-fetcher), I'd prefer to keep it here to indicate we require it.